pkg/controller/network_policy.go

```diff
@@ -1060,6 +1081,99 @@ func (cont *AciController) buildServiceAugment(subj apicapi.ApicObject,
 	}
 }
 
+func (cont *AciController) handleMulNetPolUpdate(nps []interface{}) bool {
```

Comment:
Can we take a bool argument to call WriteMulApicObjects and use the existing handleNetPolUpdate function?

Reply:
Existing handleNetPolUpdate is handling one np at a time. Here we need multiple nps at a time

Reply:
Each inner loop is the handleNetPolUpdate function. Can we not call in a loop and omit the writing to Apic part with a flag?

Reply:
Moved common code to a separate function.